INSTALL.md

```diff
@@ -139,6 +140,8 @@
 			#fastcgi_pass unix:/var/run/php5-fpm.sock;
 			fastcgi_split_path_info ^(.+\.php)(/.*)$;
 			include fastcgi_params;
+			fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
```

Comment:
Better to just use `include fastcgi.conf`instead of `include fastcgi_params`

Reply:
Yes that would works as well:

```
diff /etc/nginx/fastcgi_params /etc/nginx/fastcgi.conf --unified=0
--- /etc/nginx/fastcgi_params	2022-07-27 01:32:17.000000000 +0000
+++ /etc/nginx/fastcgi.conf	2022-07-27 01:32:17.000000000 +0000
@@ -1,0 +2 @@
+fastcgi_param  SCRIPT_FILENAME    $document_root$fastcgi_script_name;
```

Thanks for the fast reply!